site/src/@ndlib/gatsby-theme-marble/assets/i18n/json/common.es.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This is an example of adding a Spanish translation file. Translations not found in a specific language will "fallback" to whichever language is defined as the fallback in the configuration above.